skf/markdown/code_examples/java/10-code_example--encoder--.md

```diff
@@ -0,0 +1,46 @@
+
+Encoder (SQL - ESAPI)
+-----------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+package com.edw;
+
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.owasp.esapi.ESAPI;
+import org.owasp.esapi.codecs.MySQLCodec;
+
+public class encoding {
```

Comment:
@xvass could you replace the encoding class by Encoding class, since the classes should start with a Capital letter as java standard.